spec/models/recipe_spec.rb

```diff
@@ -12,4 +12,17 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe "instance methods" do
+    it "can return the total cost of a recipe's ingredients" do
+      spaghetti_with_meat_sauce = Recipe.create!(name: "Spaghetti with Meat Sauce", complexity: 3, genre: "Italian")
+      spaghetti_with_meat_sauce.ingredients.create!(name: "Tomatoes", cost: 4)
+      spaghetti_with_meat_sauce.ingredients.create!(name: "Spaghetti", cost: 5)
+      spaghetti_with_meat_sauce.ingredients.create!(name: "Ground Beef", cost: 10)
+      spaghetti_with_meat_sauce.ingredients.create!(name: "Basil", cost: 1)
+
+      total_cost = spaghetti_with_meat_sauce.total_cost
+      expect(total_cost).to eq 20
+    end
+  end
 end
```

Comment:
```ruby
def total_cost
  20
end
```

Does your assertion disprove that this is what your method looks like? Probably not. My suggestion, for instance methods, is to test more than 1 instance object so that way your code proves that the value isn't hard-coded and is dynamic.

```ruby
total_cost = spaghetti_with_meat_sauce.total_cost
expect(total_cost).to eq(20)

total_cost = spaghetti_with_butter.total_cost # add another recipe
expect(total_cost).to eq(9) 
```